test/components/Dropdown.test.js

```diff
@@ -37,49 +38,49 @@ describe('Dropdown', () => {
     expect(component).toMatchSnapshot();
   });
 
-  describe('getLocals', () => {
-
-    it('computes className', () => {
-      const { className } = dropdown.getLocals();
-      expect(className).toContain('dropdown');
-      expect(className).toContain('fancy-class-name');
-    });
+  it('computes className', () => {
+    const component = shallow(
+      <Dropdown {...exampleProps} />
+    );
+    expect(component.hasClass('dropdown')).toBe(true);
+    expect(component.hasClass('fancy-class-name')).toBe(true);
+  });
 
-    it('doesn\'t alter object value', () => {
-      const { value } = dropdown.getLocals();
-      expect(typeof value).toBe('object');
-      expect(value).toEqual(exampleProps.value);
-    });
+  it('doesn\'t alter object value', () => {
+    const component = shallow(
+      <Dropdown {...exampleProps} />
+    );
+    const value = component.prop('value');
+    expect(typeof value).toBe('object');
```

Comment:
This seems useless in TS

Reply:
it should not: `react-select` accepts numbers or strings as values, but we're forcing `Dropdown` to always pass an object. The test ensures that, when we pass a string or a number, it is correctly transformed into an object before passing it down to `react-select`

Reply:
mm, ok, but is it the case in this test?

We pass `exampleProps`, whose `value` is an object.

Reply:
mm... well, the description says `it doesn't alter object value`, so I suppose the test is meant to ensure that the value is not transformed into a string or number.

Reply:
ok, not a super interesting test then, but ok, I guess once we start using generics these problems are going away.